clio-server/config/docker-compose.yml.ctmpl

```diff
@@ -45,12 +45,12 @@ services:
       PROXY_PATH2: /api
       PROXY_URL2: http://app:{{env "CONTAINER_CLIO_PORT"}}/api
       AUTH_TYPE2: AuthType oauth20
-      OIDC_CLAIM2: Require valid-user
+      OIDC_CLAIM2: Require {{if (eq $environment "prod")}}claim user_id:{{with $picard := (vault "secret/dsde/gotc/prod/picard/picard-account.pem")}}{{$picard.Data.client_id}}{{end}}{{else}}valid-user{{end}}
```

Comment:
Note to self: it might be better to also have staging restrict access just to picard-staging, and have dev be the special case that's wide open.

Reply:
It's easy enough to just do it now

Reply:
Nevermind, will make a ticket. Restricting staging to just picard-staging is easy enough, but we also want the clio-test account to be able to access staging so it can run the integration tests post-deploy.